src/lib/clamp.js

```diff
@@ -1,47 +1,45 @@
-import React, { useState, Fragment, useEffect } from "react";
-import isCssEllipsisApplied from "./utils/isCssEllipsisApplied";
-import TruncatedElement from "./truncatedElement";
+import React, { useState, Fragment, useEffect } from 'react';
+import isCssEllipsisApplied from './utils/isCssEllipsisApplied';
+import TruncatedElement from './truncatedElement';
 
 const Clamp = ({
     children,
     lines = 2,
     maxLines = 8,
     withTooltip = true,
     withToggle = false,
-    texts = {
-        showMore: "More",
-        showLess: "Less",
-    },
-    onShowMore = () => {},
+    showMoreElement,
+    showLessElement,

```

Comment:
Consider defaulting these elements to the values you had before, so it's easier to work with out of the box?

ie:

```js
showMoreElement = showMoreButton,
showLessElement = showLessButton,
```

Reply:
Good idea thanks, I just added the default values we had before, now if you pass the withToggle prop but no values for showMoreElement and/or showLessElement those default values would be used :)

I plan to merge this changes and publish a new version today, thanks again for your support and time.

Reply:
My pleasure @mikelpmc ! Thank *you* for the great tool :)